src/interbank/interbank.service.ts

```diff
@@ -95,14 +116,52 @@ export class InterbankService {
       }
     }
     const lastPage = Math.ceil(total / pagination.size);
-
-    return formatResponse(
+    res = formatResponse(
```

Comment:
**suggestion**: const res

Reply:
I have to add two more attributes after that step. Can we new attributes to a const

Reply:
Why not
https://stackoverflow.com/questions/23436437/why-can-i-change-a-constant-object-in-javascript